distribution/lib/Standard/Base/0.0.0-dev/src/Data/Enso_Cloud/Enso_Secret.enso

```diff
@@ -87,6 +89,12 @@ type Enso_Secret
     exists name:Text parent:(Enso_File | Nothing)=Nothing =
         Enso_Secret.list parent . any s-> s.name == name
 
+    ## PRIVATE
+    to_java_value : Text | Enso_Secret | Nothing -> HideableValue
```

Comment:
Things related to security should really be `private` not just documented `## PRIVATE`, in my opinion. E.g. move the method into own `private` module, make it an extension method.

Reply:
I don't think I can do this, because `Enso_Secret.to_java_value` is used within `Standard.Database`, outside of `Standard.Base`.

I can move it to a separate file but I cannot make it inaccessible from other modules.

Note that this does not expose any 'sensitive' data - it just creates a Java representation of the `Enso_Secret` value, using its payload. It's just a transformation of existing data into an internal format.

Reply:
I have moved this method to `Enso_Cloud/Utils` and renamed it to `as_hideable_value`